admin/create-dumps.sh

```diff
@@ -136,12 +136,7 @@ DUMP_NAME=$(basename "$DUMP_DIR")
 # Backup dumps to the backup volume
 # Create backup directories owned by user "listenbrainz"
 echo "Creating Backup directories..."
-mkdir -m "$BACKUP_DIR_MODE" -p \
-         "$BACKUP_DIR/$SUB_DIR/" \
-         "$BACKUP_DIR/$SUB_DIR/$DUMP_NAME"
-chown "$BACKUP_USER:$BACKUP_GROUP" \
-      "$BACKUP_DIR/$SUB_DIR/" \
-      "$BACKUP_DIR/$SUB_DIR/$DUMP_NAME"
+mkdir -p "$BACKUP_DIR/$SUB_DIR/$DUMP_NAME"
```

Comment:
this skips the `-m "$BACKUP_DIR_MODE"` option to set the mode. Was that intended?
Note the behaviour of `-p` and `-m` here, -m only sets the permissions on the last directory:

```
alastair@bono:~/t$ mkdir -m 700 -p foo/bar/baz
alastair@bono:~/t$ ls -l
total 4
drwxrwxr-x 3 alastair alastair 4096 May 31 13:39 foo
alastair@bono:~/t$ ls -l foo/
total 4
drwxrwxr-x 3 alastair alastair 4096 May 31 13:39 bar
alastair@bono:~/t$ ls -l foo/bar/
total 4
drwx------ 2 alastair alastair 4096 May 31 13:39 baz
```

Reply:
Right, I removed this because I thought the `chmod` a few lines down would handle it.
https://github.com/metabrainz/listenbrainz-server/pull/1489/files#diff-44f9cbe734a7b984e40c16e76ad9d48859088a6efb8501f5a01c91f4e5f335e2R145

Reply:
here there is only a chmod for the files in the directory ($BACKUP_FILE_MODE) not for the directory itself ($BACKUP_DIR_MODE), however I'm not sure what the final goal of these specific permissions is, perhaps we can just leave it with the defaults in this case.